aodncore/pipeline/common.py

```diff
@@ -155,6 +156,11 @@ def get_type_from_name(cls, name):
         _, extension = os.path.splitext(name)
         return cls.get_type_from_extension(extension)
 
+    @property
+    def is_image(self):
```

Comment:
I think the concept is good. The only thing I would suggest changing here would be to rename this to to more explicit that it is asking a question about the current *type* , i.e. `is_image_type`.

`is_image` as a name overlaps with the functions that actually *do* check the contents of the file, so this could be ambiguous in terms of what is being asked.

```python
my_file_type = FileType.JPEG
is_this_an_image_type = my_file_type.is_image_type
```

Reply:
:+1: Done.